### PR TITLE
LPS-126032 - Checking if group.isStagingGroup() || group.isCompanyStagingGroup()

### PR DESCRIPTION
**Pull request by joseabelenda**

### CODE REVIEW

**Comment by natocesarrego** [comment]
Hi @joseabelenda, your pull looks good to me, although the reported test failure seems to be related, can you please double check?

Thanks.

**Comment by joseabelenda** [comment]
Yes, @natocesarrego.
I'm checking.

Thanks!

**Comment by joseabelenda** [comment]
ci:test:relevant

**Comment by joseabelenda** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/exportimport/portlet/preferences/processor/DDMFormExportImportPortletPreferencesProcessor.java [hunk 1]
```diff
@@ -126,12 +126,16 @@ public PortletPreferences processExportPortletPreferences(
 			return portletPreferences;
 		}
 
+		if (ExportImportThreadLocal.isStagingInProcess() &&
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/exportimport/portlet/preferences/processor/DDMFormExportImportPortletPreferencesProcessor.java:129]: @joseabelenda, how about reuse the previous if to attend your needs? For example:
```
if (ExportImportThreadLocal.isStagingInProcess()) {
    if (!group.isStagedPortlet(DDMPortletKeys.DYNAMIC_DATA_MAPPING_FORM_ADMIN)) {
        ...
    }

    if (!(group.isStagingGroup() || group.isCompanyStagingGroup())) {
        ...
    }
}
```

File: modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java [hunk 2]
```diff
@@ -390,7 +390,7 @@ protected void exportImportPortlet(
 						TYPE_PUBLISH_PORTLET_LOCAL,
 					settingsMap);
 
-		ExportImportThreadLocal.setPortletStagingInProcess(true);
+		ExportImportThreadLocal.setPortletExportInProcess(true);
```
Comment (natocesarrego) [inline modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java:393]: @joseabelenda, this change modifies a base class, probably, used by several other test classes, not only the Forms classes, in light of this should we really change it? After all, it'll probably affect other products. Is there any way to circumvent the test failures only changing Forms classes?

**Comment by natocesarrego** [comment]
Besides, from what I could see on CI results, the changes on the base class affected other products indeed.

**Pull request closed without merging**